cmake/Linker.cmake

```diff
@@ -0,0 +1,3 @@
+if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
```

Comment:
Shouldn't this be under an option? I think building Clio should by default require as few dependencies as possible